src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceUtils.kt

```diff
@@ -290,10 +294,14 @@ inline fun <R> whenNetworkServiceObject(
     is EvChargingUnit -> isEvChargingUnit(identifiedObject)
     is TapChangerControl -> isTapChangerControl(identifiedObject)
     is SeriesCompensator -> isSeriesCompensator(identifiedObject)
+    is SynchronousMachine -> isSynchronousMachine(identifiedObject)
```

Comment:
todo: Move to the bottom of the when to match reordered parameters.

Reply:
moved